### ISSUE TITLE
Style Color Should be Blue

### ISSUE THREAD

**Issue by jstoneCWS** [freeCodeCamp/freeCodeCamp#15469]
The style color is blue, receiving error message "should be blue"

Challenge [Use CSS Selectors to Style Elements](https://www.freecodecamp.com/challenges/use-css-selectors-to-style-elements) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36 Edge/15.15063</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<style>
  h2 {color: blue;}
</style>
CatPhotoApp</h2>

<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

```

**Comment by n-tran** [comment]
@jstoneCWS  hi, you have a syntax error. please refer to the help button. this is not a bug.

hint: your header is wrong. hope that gives you a kick start on troubleshooting.

**Comment by erictleung** [comment]
@jstoneCWS Thank you for reporting this issue. As pointed out above, you have a syntax error. The `h2` tag doesn't match over into the `<style>` tags. You still need a matching one within the body of your code.

This is a standard message notifying you that this issue seems to be a request for help. Instead of asking for help here, please click the "help" button on the challenge on freeCodeCamp, which will take you to the help chatroom for that specific challenge. You can also view our [full list of official chatrooms](http://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390).

If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding!

**Comment by jay1pimp** [comment]
On Jun 21, 2017 1:31 PM, "Eric Leung" <notifications@github.com> wrote:

Closed #15469 <https://github.com/freeCodeCamp/freeCodeCamp/issues/15469>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/freeCodeCamp/freeCodeCamp/issues/15469#event-1133445510>,
or mute the thread
